### PR TITLE
fix: removed font-medium classes as the base style for all elements is font-medium

### LINKED ISSUE #2145: Bug: Remove font-medium classes from codebase
`font-medium` is used all over the code base, but that font-weight is already said in the base styles for all elements, so it's redundant. See https://github.com/open-sauced/app/blob/beta/styles/globals.css#L11-L13.

Relates to https://github.com/open-sauced/internal-team/issues/21

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Removed redundant styling that font-medium sets.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2145

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/atoms/Checkbox/checkbox.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ const Checkbox = React.forwardRef<React.ElementRef<typeof CheckboxPrimitive.Root
         {label && (
           <label
             htmlFor={getId()}
-            className="ml-3 text-sm font-medium leading-none cursor-pointer text-light-slate-12 peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
```
Comment (nickytonline) [inline components/atoms/Checkbox/checkbox.tsx:32]: All the `font-medium` classes related to h1-h6 have been removed in all files as the base style for h1-h6 handles this now.

File: styles/globals.css [hunk 2]
```diff
@@ -9,7 +9,7 @@
 }
 
 * {
-  font-weight: 500;
+  @apply font-medium;
```
Comment (nickytonline) [inline styles/globals.css:12]: Applied the Tailwind class associated with font-weight: 500.

File: layouts/hub.tsx [hunk 3]
```diff
@@ -42,9 +42,8 @@ const HubLayout = ({ children }: { children: React.ReactNode }) => {
       }
     }
 
-    getUser()
-      .catch(console.error)
-      .then(() => {});
+    // eslint-disable-next-line no-console
+    getUser().catch(console.error);
```
Comment (nickytonline) [inline layouts/hub.tsx:46]: ESLint was erroring becaused of this. Not sure how it snuck in in previous merges.

File: styles/globals.css [hunk 4]
```diff
@@ -9,7 +9,7 @@
 }
 
 * {
-  font-weight: 500;
+  @apply font-normal;
```
Comment (nickytonline) [inline styles/globals.css:12]: @isabensusan recommended `font-weight: 400`, so using `font-normal` instead of `font-medium`.

**Review by OgDev-01 — APPROVED** [review]
LGTM

**Review by takanome-dev — APPROVED** [review]
:ship:

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**